src/main/java/uk/gov/hmcts/reform/civil/service/citizenui/MediationCSVService.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.civil.service.citizenui;
+
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.core.io.InputStreamSource;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.sendgrid.EmailAttachment;
+import uk.gov.hmcts.reform.civil.sendgrid.EmailData;
+
+import java.math.BigDecimal;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class MediationCSVService {
+
+    private final String SITE_ID = "5";
+    private final String CASE_TYPE = "1";
+    private final String CHECK_LIST = "4";
+    private final String PARTY_STATUS = "5";
+    private final String toEmail = "smallclaimsmediation@justice.gov.uk";
```

Comment:
this should not be hardcoded. this should be a keyvault value like we do with other emails